testing/btest/scripts/base/frameworks/input/setspecialcases.zeek

```diff
@@ -26,7 +26,7 @@ type Idx: record {
 
 type Val: record {
 	s: set[string];
-	s: vector of string;
+	ss: vector of string;
```

Comment:
heh.  I never noticed this one.

Reply:
Yep. The surprising things that popup when you fix a bug :smiley: